master/buildbot/test/util/integration.py

```diff
@@ -81,6 +81,14 @@ def setupConfig(self, configFunc):
                         configFunc=configFunc,
                         proto=proto,
                         workerclass=workerclass))
+
+        # mock reactor.stop (which trial *really* doesn't
```

Comment:
I plan to introduce a better fake reactor, and I think that any factorization of this code should use that fake reactor. I can't commit to adapting all the uses of this to use a fake reactor in timely fashion, though.